include/library/spdm_crypt_lib.h

```diff
@@ -428,7 +428,6 @@ void libspdm_copy_signature_swap_endian(
 #define LIBSDPM_SPDM_10_11_ASYM_VERIFY_FLAG_BIG_ENDIAN_ONLY        (0x0)
 #define LIBSPDM_SPDM_10_11_ASYM_VERIFY_FLAG_LITTLE_ENDIAN_ONLY     (0x1)
 #define LIBSDPM_SPDM_10_11_ASYM_VERIFY_FLAG_BIG_OR_LITTLE_ENDIAN   (0x2)
-#define LIBSDPM_SPDM_10_11_ASYM_VERIFY_FLAG_ENDIAN_FLAGS           (0x3)
 
```

Comment:
These are defines specifying endianness in the spdm_crypt_lib. These are different than the context but have the same values so they can be used interchangeably with the context type. See my comment about this later in the review in libspdm_com_crypto_service.c.